src/forms/assetForm.tsx

```diff
@@ -103,12 +103,11 @@ interface AssetFormState {
   loading: boolean;
   customizeModel: boolean;
 }
-function ifNullReturnEmptyString (value:string|null|undefined){
-  if (value===null || value===undefined){
-    return ""
+function ifNullReturnEmptyString(value: string | null | undefined) {
```

Comment:
same as above - changes to this file are just linting - I had a made a change in here, linted and saved, then later undid that change, so only linting is left behind